app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Adapters/DirectionsListAdapter.java

```diff
@@ -75,7 +75,7 @@ public TextView getDirectionDistance() {
             return directionDistance;
         }
 
-        public TextAdapterViewHolder(@NotNull View itemView) {
+        public TextAdapterViewHolder(View itemView) {
```

Comment:
Not saying that this is wrong but why was this changed?

Reply:
I'm not sure why, but it wasn't building on either of my computers